admin-company-service/src/helpers/constants.ts

```diff
@@ -176,12 +176,24 @@ const commonFunctions = {
     verifyToken: (token: any) => {
         try {
             const decoded = jwt.verify(token, secretKey);
+            
             return decoded;
         } catch (error) {
             // If the token is invalid or has expired, jwt.verify will throw an error
             return null;
         }
     },
+    
+    /**
+     * This method will generate hash password
+     * @param password -plain password
+     * @returns -returns encrypted password
+     */
+    hashPassword: async (password: any) => {
+        let saltRounds = 12;
+        const hashedPassword = await bcrypt.hash(password, saltRounds);
+        return hashedPassword;
+    },
```

Comment:
we should have more security of the password.
this way it can be guessed easily in 12th attempt by any intruser.

```suggestion
    hashPassword: async (password: any) => {
      const salt = randomBytes(8).toString('hex');
    const buf = (await scryptAsync(password, salt, 64)) as Buffer;

    return `${buf.toString('hex')}.${salt}`;
    },
```
something similar can be done